site/docs/v1/tech/apis/integrations.adoc

```diff
@@ -47,7 +47,9 @@ include::docs/v1/tech/apis/_integration-response-body.adoc[]
 
 == Update Integrations
 
-This API is used to update the global System Configuration object.
+:capitalized_object_name: Integration
+include::docs/v1/tech/apis/_generic-update-explanation-fragment.adoc[]
+:capitalized_object_name!:
```

Comment:
This API won't require an Id on the URL.

Reply:
Thanks, handled.